src/+otp/+arenstorf/Arenstorf.m

```diff
@@ -0,0 +1,37 @@
+classdef Arenstorf < otp.Problem
+  % The classic  3-body  problem with parameters from
+  % Reference:  Hairer, Ernst, Syvert Paul Nørsett, and Gerhard Wanner.
+  % Solving Ordinary Differential Equations I: Nonstiff Problems Springer-Verlag, 1987.
+  % CH II, p. 129.
+  methods
+    function obj = Arenstorf(timeSpan, y0, parameters)
+      obj@otp.Problem('Arenstorf Orbit Problem', 4, timeSpan, y0, parameters);
+    end
+  end
+  
+  methods (Access=protected)
+    
+    function onSettingsChanged(obj)
+      m1 = obj.Parameters.m1;
+      m2 = 1- obj.Parameters.m1;

```

Comment:
I prefer to keep m2 as a parameter.  Can this be reverted to `m2 = obj.Parameters.m2;`

Reply:
In the problem context does it make sense for m1 and m2 to not add up to 1?